docs/ingestion/data-management.md

```diff
@@ -232,11 +232,7 @@ There are other types of `inputSpec` to enable reindexing and delta ingestion.
 
 ### Reindexing with Native Batch Ingestion
 
-This section assumes the reader understands how to do batch ingestion without Hadoop using [native batch indexing](../ingestion/native-batch.md),
-which uses an `inputSource` to know where and how to read the input data. The [`DruidInputSource`](native-batch.md#druid-input-source)
-can be used to read data from segments inside Druid. Note that IndexTask is to be used for prototyping purposes only as
-it has to do all processing inside a single process and can't scale. Please use Hadoop batch ingestion for production
-scenarios dealing with more than 1GB of data.
+This section assumes you understand how to do batch ingestion without Hadoop using [native batch indexing](../ingestion/native-batch.md). Native batch indexing uses an `inputSource` to know where and how to read the input data. You can use the [`DruidInputSource`](native-batch.md#druid-input-source) to read data from segments inside Druid. Use the Index task (`index`) for prototyping purposes because it relies on a single process and can't scale. Use Parallel task (`index_parallel`) to ingest more than 1GB of data.
```

Comment:
`index_parallel` behaves almost the same as `index` when `maxNumConcurrentSubTasks` is 1. So, I think we can suggest to always use `index_parallel`, but change `maxNumConcurrentSubTasks` depending on data size.

Reply:
Thanks @jihoonson . I changed it PTAL